core/src/test/java/org/apache/calcite/sql/test/SqlOperatorBaseTest.java

```diff
@@ -9074,6 +9083,11 @@ protected void checkAggType(SqlTester tester, String expr, String type) {
         false);
     final String[] values = {"1", "2", "2"};
     tester.checkAgg("bit_or(x)", values, 3, 0);
+    final String[] binaryValues = {
+        "CAST(x'01' AS BINARY)",
+        "cast(x'02' as BINARY)",
+        "cast(x'02' AS BINARY)"};
```

Comment:
Add a test for `null` value.

Reply:
We may also need to update the `reference.md` for the function signatures change.